### PR TITLE
Add benchmarks report for TiDB v5.0

### PR DESCRIPTION
**Pull request by qiancai**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Add benchmarks report for TiDB v5.0

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: benchmark/benchmark-sysbench-v5-vs-v4.md [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+| Sysbench  | 1.0.20     |
+
+### 参数配置
+
+#### TiDB v4.0 参数配置
+
+{{< copyable "" >}}
+
+```yaml
+log.level: “error”
```
Comment (TomShawn) [inline benchmark/benchmark-sysbench-v5-vs-v4.md:38]: This is the Chinese quotation mark.
```suggestion
log.level: "error"
```

File: benchmark/benchmark-sysbench-v5-vs-v4.md [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+readpool.storage.normal-concurrency: 10
+pessimistic-txn.pipelined: true
+```
+
+#### TiDB v5.0 参数配置
+
+{{< copyable "" >}}
+
+```yaml
+log.level: “error”
```
Comment (TomShawn) [inline benchmark/benchmark-sysbench-v5-vs-v4.md:68]:
```suggestion
log.level: "error"
```

**Review by shuke987 — COMMENTED** [review]
LGTM

**Pull request merged**